codecov.yml

```diff
@@ -2,6 +2,11 @@ coverage:
   status:
     project: false
     patch: false
+ignore:
+  - packages/rmf-dashboard-framework/examples
+  - "**/*.js"
+  - "**/*.stories.tsx"
+  - "**/index.ts"
```

Comment:
`index.ts` shouldn't be ignored.

Reply:
Thanks for catching that, I accidentally left that in after trying some stuff

https://github.com/open-rmf/rmf-web/pull/1036/commits/3affe31c348d916d8f6858051c45479455a9caf6